readme.txt

```diff
@@ -113,15 +113,68 @@ Just make sure to setup your permalinks , and every thing will be fine , I promi
 6. Control plugin features from its admin page 
 
 == Changelog ==
-
 == 1.0.0 ==
 
-* Fix #137 #131 #130 #110 adaptations for wooCommerce 3.0
+Fixes #149 Enable duplication of variable products
+Fixes #165 upsets/crosssells handling in wooCommerce3
+Fixes #159 Attribute Terms synchronization issues
+
+== 0.30.RC2 WooCommerce3 new taxonomies and Attributes fixes ==
```

Comment:
Those changes are part of the version 1.0.0 now as they are merged , plus they are not published on wordpress repo , so let's make this, part of the version 1.0.0 changelog , please